tests/unit/dojson/test_dojson_hep.py

```diff
@@ -312,16 +312,16 @@ def test_licence(marcxml_to_json, json_to_marc):
 
 def test_acquisition_source(marcxml_to_json, json_to_marc):
     """Test if acquisition_source is created correctly."""
-    assert (marcxml_to_json['acquisition_source'][0]['source'] ==
-            json_to_marc['541'][0]['a'])
-    assert (marcxml_to_json['acquisition_source'][0]['email'] ==
-            json_to_marc['541'][0]['b'])
-    assert (marcxml_to_json['acquisition_source'][0]['method'] ==
-            json_to_marc['541'][0]['c'])
-    assert (marcxml_to_json['acquisition_source'][0]['date'] ==
-            json_to_marc['541'][0]['d'])
-    assert (marcxml_to_json['acquisition_source'][0]['submission_number'] ==
-            json_to_marc['541'][0]['e'])
+    assert (marcxml_to_json['acquisition_source']['source'] ==
+            json_to_marc['541']['a'])
+    assert (marcxml_to_json['acquisition_source']['email'] ==
+            json_to_marc['541']['b'])
+    assert (marcxml_to_json['acquisition_source']['method'] ==
+            json_to_marc['541']['c'])
+    assert (marcxml_to_json['acquisition_source']['date'] ==
+            json_to_marc['541']['d'])
+    assert (marcxml_to_json['acquisition_source']['submission_number'] ==
+            json_to_marc['541']['e'])
```

Comment:
If you have time it would be cool if you could convert these tests to the new style, like in https://github.com/inspirehep/inspire-next/blob/master/tests/unit/dojson/test_dojson_common_base.py.